tests/cython/PythonEvalUTest.cxxtest

```diff
@@ -297,4 +299,33 @@ public:
         // Cleanup Python.
         global_python_finalize();
     }
+
+    void testCodeBlockWithNewline()
+    {
+        // Initialize Python.
+        global_python_initialize();
+
+        AtomSpace *as = new AtomSpace();
+        PythonEval::create_singleton_instance(as);
+        PythonEval* python = &PythonEval::instance();
+
+        // Define python functions with newline within the code block
+        string result = python->eval(
+            "def fun(x):\n"
+            "    y = x + 1\n"
+            "\n"
+            "    return y\n"
+            "\n"
```

Comment:
Yeah, since opencog python interpreter is supposed to work in non-interactive/script mode as well (as is the case now) we want to support newlines within blocks.